package.json

```diff
@@ -51,19 +50,33 @@
     "@testing-library/react-hooks": "^7.0.0",
     "@types/react": "^17.0.0",
     "@types/react-dom": "^17.0.0",
+    "@typescript-eslint/eslint-plugin": "^4.0.0",
+    "eslint": "^7.0.0",
+    "eslint-config-prettier": "^8.0.0",
+    "eslint-plugin-prettier": "^3.0.0",
     "husky": "^7.0.0",
     "katex": "^0.13.0",
     "pinst": "^2.0.0",
+    "prettier": "^2.0.0",
```

Comment:
Why not v3?

Reply:
Because this PR came before v3 was released 🙂 
V3 would be good to include